contracts/ForkDelta.sol

```diff
@@ -176,35 +124,45 @@ contract ForkDelta is SafeMath {
     uint feeMakeXfer = 0;
     uint feeTakeXfer = 0;
     if (now >= freeUntilDate) {
-      feeMakeXfer = safeMul(amount, feeMake) / (1 ether);
-      feeTakeXfer = safeMul(amount, feeTake) / (1 ether);
+      feeMakeXfer = amount.mul(feeMake) / (1 ether);
+      feeTakeXfer = amount.mul(feeTake) / (1 ether);
     }
     
-    tokens[tokenGet][msg.sender] = safeSub(tokens[tokenGet][msg.sender], safeAdd(amount, feeTakeXfer));
-    tokens[tokenGet][user] = safeAdd(tokens[tokenGet][user], safeSub(amount, feeMakeXfer));
-    tokens[tokenGet][feeAccount] = safeAdd(tokens[tokenGet][feeAccount], safeAdd(feeMakeXfer, feeTakeXfer));
-    tokens[tokenGive][user] = safeSub(tokens[tokenGive][user], safeMul(amountGive, amount) / amountGet);
-    tokens[tokenGive][msg.sender] = safeAdd(tokens[tokenGive][msg.sender], safeMul(amountGive, amount) / amountGet);
+    tokens[tokenGet][msg.sender] = tokens[tokenGet][msg.sender].sub(amount.add(feeTakeXfer));
+    tokens[tokenGet][user] = tokens[tokenGet][user].add(amount.sub(feeMakeXfer));
+    tokens[tokenGet][feeAccount] = tokens[tokenGet][feeAccount].add(feeMakeXfer.sub(feeTakeXfer));
```

Comment:
Whoops, typo:

`feeMakeXfer.sub(feeTakeXfer)`

Should be

`feeMakeXfer.add(feeTakeXfer)`

Reply:
Ups.. Will fix.